src/components/Answer.tsx

```diff
@@ -166,21 +150,27 @@ export const Answer = (props: AnswerProps) => {
       <>
         {type === 'teacher-preview' && correctIncorrectIcon}
         {selectedCount}
-        {radioBox}
+        <input
+          type="radio"
+          className="answer-input-box"
+          checked={isSelected}
+          aria-checked={isSelected}
```

Comment:
Isn't this redundant with the `checked` attribute on a radio input?

Reply:
I am not sure. I tried to examine the property, `ariaChecked`, in the console when working on this and it was `null` unless I set it like this. Granted, I think that might be a disconnect specific to the javascript property. I will remove it.

Reply:
I don't think you want to look for the property, but for the state in the Accessibility tab on the far right when you examine Elements. The Computed Properties will indicate the checked state for a native checkable type.
![image](https://github.com/user-attachments/assets/c3ae4985-0be7-46ac-98ae-a56f578c38ca)